hsingyuc/hangtime
# Style workout complete page

**Features**
1. Organize scss files.
1. Add try again button to link back to the beginning of the workout.
1. Styling workout complete page.
1. Change save session icons.
1. Add history link on save session icons.

**Screenshot**
<img width="1440" alt="螢幕快照 2020-05-19 上午12 56 08" src="https://user-images.githubusercontent.com/56378160/82266607-ce055980-9972-11ea-9b05-534ca229034f.png">


**Testing**
1. Run the workout till the end and see if workout complete page rendered on completion.
1. Save the session and go to history, see if the session stored in history.
